ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -149,23 +149,25 @@ def _llmobs_set_meta_tags_from_chain(
         span.set_tag_str(SPAN_KIND, "workflow")
 
         if inputs is not None:
-            if isinstance(inputs, str):
-                span.set_tag_str(INPUT_VALUE, inputs)
-            else:
-                try:
-                    span.set_tag_str(INPUT_VALUE, json.dumps(inputs))
-                except TypeError:
-                    log.warning("Failed to serialize chain input data to JSON: %s", inputs)
+            try:
+                formatted_inputs = self.format_io(inputs)
+                if isinstance(formatted_inputs, str):
+                    span.set_tag_str(INPUT_VALUE, formatted_inputs)
+                else:
+                    span.set_tag_str(INPUT_VALUE, json.dumps(self.format_io(inputs)))
+            except TypeError:
+                log.warning("Failed to serialize chain input data to JSON: %s", inputs)
         if error:
             span.set_tag_str(OUTPUT_VALUE, "")
         elif outputs is not None:
-            if isinstance(outputs, str):
-                span.set_tag_str(OUTPUT_VALUE, str(outputs))
-            else:
-                try:
-                    span.set_tag_str(OUTPUT_VALUE, json.dumps(outputs))
-                except TypeError:
-                    log.warning("Failed to serialize chain output data to JSON: %s", outputs)
+            try:
+                formatted_outputs = self.format_io(outputs)
+                if isinstance(formatted_outputs, str):
+                    span.set_tag_str(OUTPUT_VALUE, formatted_outputs)
+                else:
+                    span.set_tag_str(OUTPUT_VALUE, json.dumps(self.format_io(outputs)))
+            except TypeError:
+                log.warning("Failed to serialize chain output data to JSON: %s", outputs)
```

Comment:
let's just remove logging the output, don't think that provides value

Reply:
fair enough, you wanna just `pass` here?

Reply:
we can do
log.warning("Failed to serialize chain output data to JSON")

Reply:
ah misread the original, sorry, makes sense